server/game/cards/01_SOR/events/SmokeAndCinders.ts

```diff
@@ -0,0 +1,27 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+
+export default class SmokeAndCinders extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '6702266551',
+            internalName: 'smoke-and-cinders',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Each player discards all but 2 cards from their hand',
+            immediateEffect: AbilityHelper.immediateEffects.simultaneous(
+                (context) => context.game.getPlayers().map((player) =>
```

Comment:
With the amount function we can simplify by not using `simultaneous()` and just making `target` an array with both players, I believe (if not then we should aim for that, I think that'd be the ideal syntax here)

Reply:
Yes, that's much better 😄